### ISSUE TITLE
Steppers 4-6 issue

### ISSUE THREAD

**Issue by gin66** [gin66/FastAccelStepper#18]
The steppers 4-6 do not work properly.

Try: `M1 A10000 V300 f M4 A10000 V300 f`

M4 runs slower than M1 and M4 makes more noise

An issue of `M4 A10000 f` or `M4 V300 f` makes stepper 4 slow down/speed up and M1 too makes noise. An issue of `M4 f` has no effect.

**Comment by gin66** [comment]
Another mystery:

Running M4 A100 V50 f

yields after some time period time of 0µs:
```
[16:27:30:567] M1: @0 M2: @0 M3: @0 M4: @74810 nonstop QueueEnd=74813/258us ACC   M5: @0 M6: @0 
[16:27:30:667] M1: @0 M2: @0 M3: @0 M4: @75240 nonstop QueueEnd=75243/258us ACC   M5: @0 M6: @0 
[16:27:30:767] M1: @0 M2: @0 M3: @0 M4: @75670 nonstop QueueEnd=75673/258us ACC   M5: @0 M6: @0 
[16:27:30:867] M1: @0 M2: @0 M3: @0 M4: @76118 nonstop QueueEnd=76124/256us ACC   M5: @0 M6: @0 
[16:27:30:967] M1: @0 M2: @0 M3: @0 M4: @76578 nonstop QueueEnd=76584/256us ACC   M5: @0 M6: @0 
[16:27:31:067] M1: @0 M2: @0 M3: @0 M4: @77038 nonstop QueueEnd=77044/255us ACC   M5: @0 M6: @0 
[16:27:31:167] M1: @0 M2: @0 M3: @0 M4: @77498 nonstop QueueEnd=77504/255us ACC   M5: @0 M6: @0 
[16:27:31:267] M1: @0 M2: @0 M3: @0 M4: @77961 nonstop QueueEnd=77964/253us ACC   M5: @0 M6: @0 
[16:27:31:367] M1: @0 M2: @0 M3: @0 M4: @78421 nonstop QueueEnd=78424/253us ACC   M5: @0 M6: @0 
[16:27:31:467] M1: @0 M2: @0 M3: @0 M4: @78881 nonstop QueueEnd=78884/251us ACC   M5: @0 M6: @0 
[16:27:31:566] M1: @0 M2: @0 M3: @0 M4: @79341 nonstop QueueEnd=79344/251us ACC   M5: @0 M6: @0 
[16:27:31:667] M1: @0 M2: @0 M3: @0 M4: @79801 nonstop QueueEnd=79804/251us ACC   M5: @0 M6: @0 
[16:27:31:766] M1: @0 M2: @0 M3: @0 M4: @80256 nonstop QueueEnd=80256/0us ACC   M5: @0 M6: @0 
[16:27:31:867] M1: @0 M2: @0 M3: @0 M4: @80706 nonstop QueueEnd=80706/0us ACC   M5: @0 M6: @0 
[16:27:31:967] M1: @0 M2: @0 M3: @0 M4: @81180 nonstop QueueEnd=81180/0us ACC   M5: @0 M6: @0 
[16:27:32:067] M1: @0 M2: @0 M3: @0 M4: @81670 nonstop QueueEnd=81670/0us ACC   M5: @0 M6: @0 
[16:27:32:167] M1: @0 M2: @0 M3: @0 M4: @82160 nonstop QueueEnd=82160/0us ACC   M5: @0 M6: @0 
[16:27:32:267] M1: @0 M2: @0 M3: @0 M4: @82650 nonstop QueueEnd=82650/0us ACC   M5: @0 M6: @0 
[16:27:32:367] M1: @0 M2: @0 M3: @0 M4: @83140 nonstop QueueEnd=83140/0us ACC   M5: @0 M6: @0 
[16:27:32:467] M1: @0 M2: @0 M3: @0 M4: @83630 nonstop QueueEnd=83630/0us ACC   M5: @0 M6: @0 
```

**Comment by gin66** [comment]
Can't get more weird:
- This works M4 A1000 V501 f
- Works too: V501 U
- Doesn't work: V500 U
   This leads to approx 6ms long pulses with 500µs period, followed by 4 ms without pulses.
    10ms is the wait time of the periodic Stepper task

**Comment by gin66** [comment]
V501 is the last value, where the ramp generator generates single step commands.
For V500 and lower two steps will be generated
=> Apparently for the 2 steps command only one will be generated and the queue runs out of commands within 10ms.

Funny. 

Stopping M4 with S, lead to a problem with M1. Stopping M4 with X (and out of reset), there is no problem

**Comment by gin66** [comment]
Combination M1/M5 and M1/M6 do not show the stepper problem spill over. M2/M5 has the issue.

Difference between forceStop() and normal stop: forceStop() does not disable the interrupt.

**Comment by gin66** [comment]
Fixed in V0.13.2

nextCommand() has used mcpwm timer index (0-2) to change pulse counter setting, which requires 0-5.
This is OK for stepper 0-2, but not ok for stepper 3-5
